lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -775,12 +775,15 @@ tokenize_number(Rest, Acc, false) ->
   {Rest, list_to_integer(lists:reverse(Acc)), length(Acc)}.
 
 tokenize_hex([H|T], Acc) when ?is_hex(H) -> tokenize_hex(T, [H|Acc]);
+tokenize_hex([$_|T], Acc) -> tokenize_hex(T, Acc);
```

Comment:
We need to make sure the next item is a digit. So it should be something like:

    tokenize_hex([$_,H|T], Acc) when ?is_hex(H) -> tokenize_hex(T, [H|Acc]);

Can you please amend this clause and the other ones?